app/components/shared/widget-inputs/WTextAreaInput.vue

```diff
@@ -1,11 +1,18 @@
 <template>
-  <textarea
+  <span>
+    <textarea
       type="text"
       :placeholder="metadata.placeholder"
       :value="value"
       @input="emitInput($event.target.value)"
       class="text-area radius"
-  />
+      :name="uuid"
+      v-validate="validate"
+    />
+    <span class="input-error" v-show="errors.first(uuid)">
+      {{ errors.first(uuid) }}
+    </span>
+  </span>
```

Comment:
Should check if this is going to conflict with the widget input refactor @holiber is currently doing.